airgun/widgets.py

```diff
@@ -1146,16 +1163,17 @@ def read(self):
         return read_rows
 
 
-class SatContentCountsTable(SatTable):
-    """'Content Counts' table present in every repository, containing amount of
-    packages/puppet modules/etc with links to corresponding details pages.
+class SatTableWithUnevenStructure(SatTable):
```

Comment:
Maybe we should we try looking at SummaryTable from manageiq first? Or you prefer to refactor that later?

Reply:
I am still in doubt that we need that extra logic and if we need it, we should improve it, but anyway, let's do it in other PR

Reply:
Imo extra logic is defining our own tables for widget which is already automated and (hopefully) can be easily imported, but i'm ok with separate PR later.